### ISSUE TITLE
NTR: Cryptococcus in blood, urine, or sputum culture positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8105]
New term request Cryptococcus in blood, urine, or sputum culture positive (nadea3_190725193345): Cryptococcal  UMLS: 2748377

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000906
- label: Positive bloodstream cryptococcus culture
- parents: [Fungal bloodstream infection (HP:0034749)](https://hpo.jax.org/app/browse/term/HP:0034749)
- pmid: 
- def: Growth of Cryptococcus in a culture derived from a blood sample.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8105
- synonyms:
